packages/lesswrong/server/resolvers/revisionResolvers.ts

```diff
@@ -162,3 +162,35 @@ augmentFieldsDict(Revisions, {
     }
   },
 })
+
+defineQuery({
```

Comment:
Adding a new resolver for converting documents between different editor formats(?)